### PR TITLE
Add Flow CR documentation

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

File: docs/modules/ROOT/pages/flow.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+== Run Flow
+
+To run Flow on Kubernetes using Hazelcast Platform Operator you need to create a Flow CR.
+
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/flow.yaml[]
+----
+
+Additional flow configuration can be accheived by providing additional parameters via `OPTIONS` environment variable.
```
Comment (amandalindsay) [inline docs/modules/ROOT/pages/flow.adoc:50]:
```suggestion
Additional flow configuration can be achieved by providing additional parameters using the `OPTIONS` environment variable.
```

File: docs/modules/ROOT/pages/flow.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
+= Deploy Flow
+
+:description: In this tutorial, you'll deploy a Flow application using Hazelcast Platform Operator for Kubernetes.
+
+{description}
+
+== Prerequisites
+
+- Flow requires the Management Center application to run and be configured properly. The Management Center application should be configured to use FLow.
+
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/flow.adoc:10]: Please mention FLOW MC image should be used here --> `docker.io/hazelcast/management-center-flow`

**Review by hasancelik — CHANGES_REQUESTED** [review]

**Review by amandalindsay — CHANGES_REQUESTED** [review]
Minor copy edit, otherwise looks good

File: docs/modules/ROOT/pages/flow.adoc (hunk 2 above)
Comment (SeriyBg) [reply docs/modules/ROOT/pages/flow.adoc:10]: I have added it to the example config with the following description

File: docs/modules/ROOT/pages/flow.adoc [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+----
+...
+image: docker.io/hazelcast/management-center-flow <1>
+...
+hazelcastClusters:
+- address: flow <2>
+  name: flow <3>
+jvm:
+  args:
+  - "-Dhazelcast.mc.flow.addresses=http://233.252.0.158/" <4>
```
Comment (pollett) [inline docs/modules/ROOT/pages/flow.adoc:23]: Not sure if you commonly use this elsewhere, but this ip in the examples looks slightly strange to me, it being ASM reserved makes it look super specific, but I presume it's just a placeholder, could perhaps replace with a placeholder FQDN instead?

Comment (SeriyBg) [reply docs/modules/ROOT/pages/flow.adoc:23]: Well, I specifically used the IP range reserved for documentation purposes as a placeholder. In the description, I have explained that it should be publicly available one 😄

File: docs/modules/ROOT/pages/flow.adoc [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+...
+----
+
+<1> Management Center Flow image.
+<2> The service name of the Flow application. The same as the Flow Custom Resource (CR) name.
+<3> Hazelcast cluster name. Should always be `flow`.
+<4> Flow Ingress public address.
+<5> The internal address of Flow in the format of `http://<service-name>:9021`. The service name is the same as the Flow CR name.
+
+- To connect to the database you need create a `Secret` with the DB credentials:
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/flow.adoc:34]: The DB have to be Postgre so we should mention in here or at the beginning of the section. WDYT?

Comment (SeriyBg) [reply docs/modules/ROOT/pages/flow.adoc:34]: I thought we didn't mention it on purpose, don't we?

**Comment by hasancelik** [comment]
I think we should also mention about license usage also. Users can not use their regular license key but they can use their flow specific license key for the flow, and MC flow via creation one k8s secret. WDYT?

File: docs/modules/ROOT/pages/flow.adoc (hunk 4 above)
Comment (hasancelik) [reply docs/modules/ROOT/pages/flow.adoc:34]: Why?

If flow mentioned about it we can give reference to it also

Comment (SeriyBg) [reply docs/modules/ROOT/pages/flow.adoc]: Done

**Review by amandalindsay — APPROVED** [review]

**Review by amandalindsay — APPROVED** [review]

**Pull request merged**